src/ui/components/ShowMoreCard/styles.scss

```diff
@@ -56,7 +56,9 @@
 
 .PermissionsCard {
   .Card-contents {
-    border-radius: 0;
+    // FIX FOR ISSUE 9551
```

Comment:
we don't need a code comment unless it's a particularly obscure fix (and this isn't)

Reply:
Okay Sir, understood!
I have removed the comment.